media_kit/lib/src/platform_player.dart

```diff
@@ -101,6 +101,14 @@ class PlayerConfiguration {
   /// Default: `null`.
   final String? title;
 
+  /// This controls how much the demuxer is allowed to buffer ahead in bytes.
```

Comment:
nit: improve the documentation.
a simple note explaining usage & the backend this parameter will work on is enough.

```dart
/// Sets the demuxer cache size (in bytes) for libmpv backend.
///
/// Default: `32` MB or `32 * 1024 * 1024` bytes.
```

Also, may you please change `on` to `for` in all these comments:

https://github.com/alexmercerind/media_kit/blob/20137328df02716ecdd984313fb81c17bc1809f6/media_kit/lib/src/platform_player.dart#L67
https://github.com/alexmercerind/media_kit/blob/20137328df02716ecdd984313fb81c17bc1809f6/media_kit/lib/src/platform_player.dart#L71
https://github.com/alexmercerind/media_kit/blob/20137328df02716ecdd984313fb81c17bc1809f6/media_kit/lib/src/platform_player.dart#L75
https://github.com/alexmercerind/media_kit/blob/20137328df02716ecdd984313fb81c17bc1809f6/media_kit/lib/src/platform_player.dart#L79
https://github.com/alexmercerind/media_kit/blob/20137328df02716ecdd984313fb81c17bc1809f6/media_kit/lib/src/platform_player.dart#L83
https://github.com/alexmercerind/media_kit/blob/20137328df02716ecdd984313fb81c17bc1809f6/media_kit/lib/src/platform_player.dart#L99